tools/metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "StableReleaseTag": "v7.0.3",
-  "PreviewReleaseTag": "v7.1.0-preview.7",
+  "PreviewReleaseTag": "v7.1.0-rc.1",
```

Comment:
```suggestion
  "PreviewReleaseTag": "v7.1.0-preview.7",
```

Reply:
How about the rest keys, especially `NextReleaseTag`? Should any of them be updated?